capstone/src/main/java/com/google/sps/service/FilterService.java

```diff
@@ -10,26 +10,27 @@
 import com.google.appengine.api.datastore.Query.FilterPredicate;
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.api.datastore.Key;
-import com.google.appengine.api.datastore.KeyFactory.Builder;
+import com.google.appengine.api.datastore.KeyFactory;
 
 public class FilterService {
 
     private static final String KEY_PROPERTY = "__key__";
 
-    public static Entity getEntity(String entity, Long id) {
+    public static Entity getEntity(String entity_name, Long id) {
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-        Key key = new Builder(entity, id).getKey();
+        Key key = new KeyFactory.Builder(entity_name, id).getKey();
         Filter filter = new FilterPredicate(KEY_PROPERTY, FilterOperator.EQUAL, key);
-        Query query = new Query(entity).setFilter(filter);
+        Query query = new Query(entity_name).setFilter(filter);
         PreparedQuery result = datastore.prepare(query);
         return result.asSingleEntity();
     }
 
-    public static PreparedQuery getQuery(String entity, String property, Key key) {
+    public static PreparedQuery getQueryOfKey(String entity_name, String property, Key key) {
```

Comment:
This is still not particularly informative. Maybe call it getEntityQueryByKeyedProperty